### PR TITLE
devel/R-cran-whisker:0.4

### PR DESCRIPTION
**Pull request by yzgyyang**

### CODE REVIEW

File: patches/devel.R-cran-whisker.0.4.diff [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
++# Created by: Guangyuan Yang <ygy@FreeBSD.org>
++# $FreeBSD$
++
++PORTNAME=	whisker
++DISTVERSION=	0.4
++CATEGORIES=	devel
++DISTNAME=	${PORTNAME}_${DISTVERSION}
++
++MAINTAINER=	ygy@FreeBSD.org
++COMMENT=	{{mustache}} for R, Logicless Templating
```
Comment (lwhsu) [inline patches/devel.R-cran-whisker.0.4.diff:28]: `{{mustache}}` looks a bit weird, is that intentional?

Comment (yzgyyang) [reply patches/devel.R-cran-whisker.0.4.diff:28]: Yup that is how the mustache template looks like haha! I could change it to normal if you feel like this is too much XD

Comment (lwhsu) [reply patches/devel.R-cran-whisker.0.4.diff:28]: I'm fine If it shows fine in pkg's metadata.

**Review by lwhsu — APPROVED** [review]

File: patches/devel.R-cran-whisker.0.4.diff (hunk 1 above)
Comment (yzgyyang) [reply patches/devel.R-cran-whisker.0.4.diff:28]: I will actually change it to `Logicless mustache templating for R` just to be safe.

**Pull request merged**